### ISSUE TITLE
Jenkins issues re: Python 3.9

### ISSUE THREAD

**Issue by caufieldjh** [Knowledge-Graph-Hub/kg-obo#177]
After changing over the Jenkins build to a new Docker image with Python 3.9.14, builds fail with:
```
Error: Command '['/var/lib/jenkins/workspace/knowledge-graph-hub_kg-obo_main/gitrepo/venv/bin/python3.9', '-Im', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1.
```

Initially, this just seems like a missing `venv` installation, but after `apt-get install python3.9-dev python3.9-venv` the issue is still unresolved.

Running `python3.9 -Im ensurepip --upgrade --default-pip` on its own from within the Docker container completes as expected:
```
Looking in links: /tmp/tmpwwcx5m_0
Requirement already satisfied: setuptools in /usr/lib/python3/dist-packages (45.2.0)
Processing /tmp/tmpwwcx5m_0/setuptools-58.1.0-py3-none-any.whl
Requirement already satisfied: pip in /usr/local/lib/python3.9/dist-packages (22.2.2)
Installing collected packages: setuptools
  Attempting uninstall: setuptools
    Found existing installation: setuptools 45.2.0
    Uninstalling setuptools-45.2.0:
      Successfully uninstalled setuptools-45.2.0
Successfully installed setuptools-58.1.0
```

**Comment by caufieldjh** [comment]
Hrm, Jenkins wants to run `venv` from `/var/lib/jenkins/workspace/knowledge-graph-hub_kg-obo_main/gitrepo/venv/bin/python3.9` so maybe that's the issue

**Comment by caufieldjh** [comment]
There's also the likely possibility that the Jenkins run is just using a cached Docker image rather than an updated one

**Comment by caufieldjh** [comment]
Fixes by #178
